inference-engine/tests/functional/plugin/conformance/subgraphs_dumper/include/matchers/matchers_manager.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "pugixml.hpp"
+#include "ngraph/node.hpp"
+#include "single_op.hpp"
+
+namespace SubgraphsDumper {
+class Matcher;
+
+class MatchersManager {
+public:
+    typedef std::map<std::string, std::function<Matcher::Ptr()>> registry_map;
```

Comment:
1) `using` is more modern way to get an [alias](https://en.cppreference.com/w/cpp/language/type_alias).
2) IMHO types and alias to types should be CamelCase